#!/usr/bin/env python
PACKAGE = "pladypos"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("enableRevs", bool_t, 0, "Enable revolution control.",False)

for i in range(4):
 gen.add("rev"+str(i), int_t, 0, "Revolutions", 0, -255, 255)

gen.add("lock12", bool_t, 0, "Lock thrusters P0 and P1", True)
gen.add("lock34", bool_t, 0, "Lock thrusters P2 and P3", True)

exit(gen.generate(PACKAGE, "pladypos_node", "ThrusterMapping"))